expat/lib/expat.h

```diff
@@ -174,8 +174,10 @@ struct XML_cp {
 };
 
 /* This is called for an element declaration. See above for
-   description of the model argument. It's the caller's responsibility
-   to free model when finished with it.
+   description of the model argument. It's the user code's responsibility
+   to free model when finished with it. See XML_FreeContentModel().
```

Comment:
With my picky hat on: My vote for dropping the `()` here^^ (because the call will have 2 rather than 0 parameters).